react/components/Notification/notificationBar.js

```diff
@@ -1,26 +1,24 @@
 import React, { memo } from 'react'
-import { string } from 'prop-types'
-import NotificationContent from './notificationContent'
+import { string, object } from 'prop-types'
```

Comment:
I would go and change this to use props like it's usually it's used

```suggestion
import PropTypes from 'prop-types'
```